spec/models/book_spec.rb

```diff
@@ -6,6 +6,11 @@
     it { should validate_presence_of(:author) }
 
     subject { FactoryGirl.build(:book) }
-    it { should validate_uniqueness_of(:title).case_insensitive}
+    it { should validate_uniqueness_of(:title).case_insensitive }
+  end
+
+  describe 'associations' do
+    it { should have_many(:checkouts) }
```

Comment:
Thanks for the advice @derekprior. If you don't mind, I am going to keep them for now. Once I get a better hang of TDD, I will start to remove them.